features/networking/sdn.feature

```diff
@@ -358,3 +358,36 @@ Feature: SDN related networking scenarios
     | nmcli \| grep veth \| wc -l |
   And evaluation of `@result[:response].split("\n")[0]` is stored in the :no_of_veths clipboard
   Then the expression should be true> cb.no_of_veths==cb.no_of_unmanaged_veths
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-29299
+  @admin
+  Scenario: Without allow the migration operation, migration cannot be executed
+    When I run the :annotate client command with:
+       | resource     | network.operator.openshift.io                       |
+       | resourcename | cluster                                             |
+       | keyval       | networkoperator.openshift.io/network-migration=true |
+    Then the step should fail
+    And the output should contain:
+      | networks.operator.openshift.io "cluster" is forbidden |
+    Given I switch to cluster admin pseudo user
+    And as admin I successfully merge patch resource "network.config.openshift.io/cluster" with:
+      | {"spec":{"networkType":"OVNKubernetes"}}  |
```

Comment:
@huiran0826 Before trying to migrating to OVN, should test case make sure the origin networktype running in the cluster is "OpenShiftSDN"?

Reply:
@weliang1  Thanks,  in the test case we already has the network backbend as  OpenShiftSDN.